StompConnectionFactory.php

```diff
@@ -71,15 +74,14 @@ public function __construct($config = 'stomp:')
      */
     public function createContext(): Context
     {
-        $useExchangePrefix = self::SCHEME_EXT_RABBITMQ === $this->config['target'] ? true : false;
-
         if ($this->config['lazy']) {
-            return new StompContext(function () {
-                return $this->establishConnection();
-            }, $useExchangePrefix);
```

Comment:
I've removed `$useExchangePrefix` as it can be calculated in the constructor of `StompContext` now based on the information it gets from the current extension type.